### DISCUSSION TITLE
Recomended firmware version

### DISCUSSION THREAD

**Discussion by rollingafull** [Anime4000/RTL960x#223] [General]
Hi!

I'm working ODI stick with M110_sfp_ODI_220304.tar firmware version, and i'm thinking upgrade it. Sometimes my rb5009 doesn't link correctly to 2.5g base-T.
Can new firmware versions improve this or other functionalities?
What version do you recommend I install?

Regards.

**Comment by Anime4000** [comment]
You not suppose to use `2.5G baseT` since PON Stick is **Fiber** not Twisted Pair

You should do this instead
![image](https://github.com/Anime4000/RTL960x/assets/1908715/d85f4991-7ad4-4f43-87fa-e917075ea703)

Then Telnet into ODI Stick and do this:
```
flash set LAN_SDS_MODE 6
```

I been using that way and RB5009 connect ODI much faster and no delay on LAN SDS Auto Nego

**Reply by rollingafull** [reply]
Ok, i'll try It. And upgrading the firmware versión, can i improve anything?

Regards.

**Reply by Anime4000** [reply]
just upgrade your mikrotik firmware to 7.12,1+
then `flash set LAN_SDS_MODE 6`

**Reply by rollingafull** [reply]
Oh yes, i'm not explain correctly sorry. I refer to a stick firmware versión. Witch do you think is the best, i'm on 220304.

On RB5009 i'm on 7.13 now.

Thanks un advance.

**Reply by Anime4000** [reply]
I use 220923, works well with my ISP

**Reply by rndm2** [reply]
Can confirm 220923 is the latest optimal firmware. It still ignores VLAN map table from OLT, but it works with both ethernet and multicast traffic

**Reply by Anime4000** [reply]
the `220923` is mainly for Bridge, it's ignore MIB 171 (OLT VLAN 123 ➡️ ONT VLAN 321), this mean it forward OLT VLAN side only, if want firmware respect MIB 171, get HGU firmware instead

**Reply by rndm2** [reply]
Btw, HGU firmware also will ignore MIB 171.

**Reply by Anime4000** [reply]
that's odd, it worked on other place.
may I know what OLT currently is?

**Reply by rndm2** [reply]
Alcatel
My OLT is strange one - I don't need PLOAM to authentificate

**Reply by arag00rn** [reply]
I've a RB5009 (7.13.3) and LTE3415-SH+

I've set LAN_SDS_MODE=6

Unfortunately even with auto neg off and speed 2.5G baseX my link is always down.
Instead if I set auto neg ON and 1G baseX, connection works fine.

This is what I see via telnet
```
# /flash get LAN_SDS_MODE
LAN_SDS_MODE=6
# cat /proc/lan_sds/lan_sds_cfg
lan_sds_mode = 1(Fiber 1G)
```

**Reply by Anime4000** [reply]
`LTE3415-SH+` using RTL9601CI which is don't have 2500Base-X like V2801F,
You need to set `4` (HiSGMII PHY), Mikrotik put HiSGMII on `2.5G baseX`

**Reply by arag00rn** [reply]
Solved. Issue was wrong bootloader :)

**Reply by nickjack93** [reply]
hello @arag00rn , could you explain me how fix the link down problem, please?, i dont know what you refer about "wrong bootloader". Please Help Me.. thanks
